src/main/java/core/basesyntax/service/impl/RecordDataImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.RecordData;
+import java.util.Map;
+
+public class RecordDataImpl implements RecordData {
+    private static final String SEPARATOR = ",";
+
+    @Override
+    public String recorder(Map<String, Integer> fruits) {
+        String header = "fruit, quantity";
+        StringBuilder builder = new StringBuilder();
+        builder.append(header).append(System.lineSeparator());

```

Comment:
you can .append directly to constructor
builder is a bad name, don't name after data types, it doesn't add any useful information